app/packages/files/file.ts

```diff
@@ -120,7 +120,12 @@ export default abstract class File {
     @task
     @waitFor
     async getRevisions() {
-        const responseObject = await fetch(`${this.links.upload}?revisions=&`);
+        const revisionsLink = new URL(this.links.upload as string);
+        const params = revisionsLink.searchParams;
+        params.set('revisions', '');
+        revisionsLink.search = params.toString();
```

Comment:
I think the following is enough.
```suggestion
        responseLink.searchParams.set('revisions', '');
```
Is there a reason why we need to assign `params.toString()` back to `revisionsLink.search` again?

Reply:
I had some bad docs I was following. This is much better, thank you.